pg_search/src/postgres/customscan/pdbscan/privdat.rs

```diff
@@ -38,7 +38,11 @@ pub struct PrivateData {
     var_attname_lookup: Option<FxHashMap<(Varno, pg_sys::AttrNumber), String>>,
     maybe_ff: bool,
     segment_count: usize,
-    which_fast_fields: Option<Vec<WhichFastField>>,
+    // The fast fields which were identified during planning time as potentially being
+    // needed at execution time. In order for our planning-time-chosen ExecMethodType to be
+    // accurate, this must always be a superset of the fields extracted from the execution
+    // time target list.
+    which_fast_fields: Option<FxHashSet<WhichFastField>>,
```

Comment:
Please use a different name for `which_fast_fields ` (e.g., `which_fast_fields_planning`) not to confuse it with the same field name under `FastFieldExecState `